Formula/v/vineflower.rb

```diff
@@ -0,0 +1,32 @@
+class Vineflower < Formula
+  desc "Java decompiler"
+  homepage "https://vineflower.org/"
+  url "https://github.com/Vineflower/vineflower/archive/refs/tags/1.11.1.tar.gz"
+  sha256 "104df4042023190416bff2ad0e2386ddf7669c6aa585f19b9d965a7f5ca5132b"
+  license "Apache-2.0"
+
+  depends_on "openjdk@21"
+
+  def install
+    system "./gradlew", "build", "-x", "test"
```

Comment:
can use gradle formula to do the build?

```suggestion
    system "gradle", "build", "-x", "test"
```

Reply:
I'm not sure. The project wants Gradle 8.10, which is fetched automatically through the `gradlew` wrapper script. Other Gradle 8s might work, like in the `gradle` formula. I'll try this.

Reply:
The `brew audit` will complain if I try to set `gradle@8` instead of `gradle` plainly because `gradle@8` is an alias. The formula will probably not work when `gradle` is updated. Is that a problem?

Reply:
Other formulae do this with Gradle so I assume it is not